### ISSUE TITLE
Unable to process payouts:

### ISSUE THREAD

**Issue by stone212** [sammy007/open-ethereum-pool#294]
#### System information

Geth version: Parity 1.8.8

Redis version: 3.0.6

OS & Version: Ubuntu 16.04

Commit branch and hash: master

#### Expected behaviour

Payouts should process as specified in payouts.json

#### Actual behaviour

Payouts had worked for some time.  Now I get this error:

`2018/02/07 09:36:24 Unable to process payouts: Post http://127.0.0.1:8545: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
`
#### Steps to reproduce the behaviour

I can confirm that the author account is correct and I am prompted for the password to unlock it at the interval specified as payout interval in payouts.json.   127.0.0.1:8545 is the correct RPC port and it responds correctly to RPC requests.  Here is an example of a curl request that works fine:

```
curl -H "Content-Type: application/json" -X POST --data '{"jsonrpc":"2.0","method":"web3_clientVersion","params":[],"id":67}' 127.0.0.1:8545
{"jsonrpc":"2.0","result":"Parity//v1.8.8-stable-143d484-20180201/x86_64-linux-gnu/rustc1.23.0","id":67}
```

Here is code from payouts.json (which worked previously).  

```
"redis": {
                "endpoint": "127.0.0.1:6379",
                "poolSize": 10,
                "database": 0,
                "password": ""
        },

        "unlocker": {
                "enabled": true,
                "poolFee": 0,
                "poolFeeAddress": "",
                "donate": false,
                "depth": 120,
                "immatureDepth": 20,
                "keepTxFees": false,
                "interval": "10m",
                "daemon": "http://127.0.0.1:8545",
                "timeout": "10s"
        },

        "payouts": {
                "enabled": true,
                "requirePeers": 5,
                "interval": "15m",
                "daemon": "http://127.0.0.1:8545",
                "timeout": "10s",
                "address": "0x....",
                "gas": "21000",
                "gasPrice": "50000000000",
                "autoGas": true,
                "threshold": 50,
                "bgsave": false
        }

}

```

**Comment by atlanticcrypto** [comment]
I've begun having the same issue with any Parity version > 1.8.6. I've also had issues with peer counts on clients > 1.8.6, so I've downgraded my Parity instances and the problem is solved for now. Not a solution, but it's seemingly a Parity problem rather than a Sammy007 problem.

**Comment by stone212** [comment]
To be sure, is it > 1.8.6 or >= 1.8.6?  What I mean is did a downgrade to 1.8.6 solve the problem?

**Comment by atlanticcrypto** [comment]
A downgrade to 1.8.6 solved the problem for me.

**Comment by stone212** [comment]
Confirmed. Then the solution will be to find out what changed in Parity to block payouts from working and then to see what code we have to modify in open-ethereum-pool to resolve this issue.

**Comment by stone212** [comment]
@Njcrypto Does this still work for you?  I still get this error when using Parity 1.8.6 now.

**Comment by stone212** [comment]
This problem has resolved with Parity 1.10.9.  Closing issue.
